_api-reference/cluster-health.md

```diff
@@ -105,4 +105,112 @@ The following table lists all response fields.
 ## Required permissions
 
 If you use the security plugin, make sure you have the appropriate permissions:
-`cluster:monitor/health`.
\ No newline at end of file
+`cluster:monitor/health`.
+
+## Cluster health by awareness attribute
+Introduced 2.5
+{: .label .label-purple }
+
+Cluster health by awareness attribute is an experimental feature. Therefore, we do not recommend the use of cluster health by awareness attribute in a production environment. For updates on the progress of cluster health by awareness attribute, or if you want leave feedback that could help improve the feature, refer to the [GitHub issue]().
```

Comment:
This is where our style really falls flat, in my opinion. "Cluster health by awareness attribute is ..." The way we name and style these features allows a feature name (especially a long-winded one like this) to get lost in text. It's awkward to read it in a sentence. We should have a way of setting out the feature name. But that's a conversation and not a topic for review, I suppose.    

"... or if you want **to** leave feedback that could help improve the feature, refer to the [GitHub issue]()."
Also need the url for this link.

Reply:
I agree with Chris. @natebower - can we include the feature name in quotes, for example, "The "cluster health by awareness" attribute is an ...."?

Reply:
I also agree, but can we do better than quotes?

Reply:
How about bolding the name of the feature?

Reply:
How about, "We are introducing a new attribute called _cluster health by awareness_. We do not recommend using the attribute in a production environment.

Reply:
It's not an attribute though, it's a change to the existing API. It's how it returns the response.

Reply:
I am waiting for the issue link from the developer. Will put the link in once he provides it.

Reply:
@hdhalter To clarify: The name is "[cluster health] by [awareness attribute]". Cluster health is an API, and awareness attribute is a term.